BackendServices/AuxiliaryServices/WebAPIService/CCPGames/Dust514Class.cs

```diff
@@ -0,0 +1,350 @@
+                            }
+                            break;
+                        case "/cfg/":
+                            if (contentTypeValue == "application/vnd.ccp.eve.RawConfiguration-v1+json")
+                            {
+                                return $"{{\"allowUserCorporation\": 0 }}";
+                            }
+                            break;
+                        default:    
+                            break;
+                    }
+                    break;
+                case "PUT":
+                    switch (absolutepath)
+                    {
+                        case "/":
+                            return @"{""configuration"":{""href"":""http://dust514.online:26004/cfg/""}}";
+                            
+                    }
+                    break;
+                case "POST":
+                    switch (absolutepath)
+                    {
+                        case "/dust/auth/":
+                            
+                            string pattern = @"character=(\d+)";
+                            string authData = Encoding.UTF8.GetString(PostData);
+                            Match match = Regex.Match(authData, pattern);
+
+                            if (match.Success)
+                            {
+                                GlobalState.charaId = match.Groups[1].Value;
+                                LoggerAccessor.LogWarn("Character ID: " + GlobalState.charaId);
+                                GlobalState.isLoggedIn = true;
+                            }
+                            else
+                            {
+                                LoggerAccessor.LogWarn("Character ID not found.");
+                                GlobalState.isLoggedIn = false;
+                            }
+
+                            return "{\"access_token\":\"8d00eee7-0405-5b40-de54-218d79cc17ce\",\"refresh_token\":\"8d00eee7-0405-5b40-de54-218d79cc17ce\",\"token_type\":\"dust\"}";
+                    }
+                    break;
+                default:
+                    break;
+            }
+
+            if (absolutepath.Equals($"/corporation/1/member/{GlobalState.charaId}/]"))
```

Comment:
So, a typical pitfall where I am angry towards Microsoft : .Equals() not check the validity of the string input, doing == instead fixes potential deadlocks at compile and will not make the server crash on invalid requests.